Sources/SwiftGRPC/Core/CallResult.swift

```diff
@@ -41,7 +41,7 @@ public struct CallResult: CustomStringConvertible {
     trailingMetadata = op.receivedTrailingMetadata()
   }
   
-  fileprivate init(success: Bool, statusCode: StatusCode, statusMessage: String?, resultData: Data?,
+  public init(success: Bool, statusCode: StatusCode, statusMessage: String?, resultData: Data?,
```

Comment:
Please add a comment explaining that this method is only public for use by test stubs.